PythonRegression/tests/features/steps/blowball_tests_steps.py

```diff
@@ -9,25 +9,25 @@
 
 tests = api_test_logic
 
-config = {}
-responses = {}
-test_vars = {}
+world.config = {}
+world.responses = {}
+world.test_vars = {}
```

Comment:
you again have a PR that does 2 things. Add API calls, and start using `world`... (refrain from dong this next time. Use separate PRs. It is supposedly fine that you started to use `world` in the api tests in this PR, but this file is out of scope)
If you start using world, you should have a hook that cleans it after each feature is complete.